modules/ROOT/pages/clustering/unbind-system-database.adoc

```diff
@@ -0,0 +1,83 @@
+:description: The page describes the neo4j-admin command `dbms unbind-system-db`.
+[role=enterprise-edition]
+
+= Unbind the `system` database
+
+You can use the `neo4j-admin dbms unbind-system-db` command to remove and archive the cluster state for the `system` database, so that the server can rebind to a new `system` database in the cluster.
+
+[CAUTION]
+====
+Executing the `neo4j-admin dbms unbind-system-db` command does not affect the cluster state of other standard databases.
+However, since all servers in the DBMS need to be shut down to run the command, downtime will occur for all databases inside of the DBMS.
+====
+[[unbind-system-db-syntax]]
+== Syntax
+
+The `neo4j-admin dbms unbind-system-db` command has the following syntax:
+
+----
+neo4j-admin dbms unbind-system-db [-h] [--expand-commands] [--verbose] [--archive-cluster-state[=true|false]]
+                                  [--additional-config=<file>] [--archive-path=<path>]
+----
+
+
+[[unbind-system-db-description]]
+== Description
+
+Removes and archives cluster state for `system` database.
+
+[[unbind-system-db-options]]
+== Options
+
+
+The `neo4j-admin dbms unbind-system-db` command has the following options:
+
+.`neo4j-admin dbms unbind-system-db` options
+[options="header", cols="5m,6a,2m"]
+|===
+| Option
+| Description
+| Default
+
+|--additional-config=<file>
+|Configuration file with additional configuration.
+|
+
+|--archive-cluster-state[=true\|false]
+|Enable or disable the cluster state archiving.
+|false
+
+|--archive-path=<path>
+|Destination (file or folder) of the cluster state archive.
+|
+
+|--expand-commands
+|Allow command expansion in config value evaluation.
+|
+
+|-h, --help
+|Show this help message and exit.
+|
+
+|--verbose
+|Enable verbose output.
+|
+|===
+
+[[unbind-system-db-limitations]]
+== Limitations
+
+The Neo4j process must be shut down on all servers before running the `neo4j-admin dbms unbind-system-db` command.
+
+[[unbind-system-db-usage]]
+== Usage
+
+The `neo4j-admin dbms unbind-system-db` command removes and archives cluster state for the `system` database, so that the server can rebind to a new `system` database in a cluster.
+
+This command should be run on all remaining members of a cluster to work as expected.
```

Comment:
This is my whole suggestion (all comments here combined), if you want to apply it:

```suggestion
You must shut down all cluster members before running the command on each of them.
For details, see xref:clustering/multi-region-deployment/disaster-recovery.adoc#make-the-system-database-write-available[Disaster recovery guide -> Make the `system` database write-available].

The `neo4j-admin dbms unbind-system-db` command does not affect the cluster state of the standard databases.
However, since all servers in the DBMS must be shut down when the command is issued, downtime is expected for all databases within the DBMS.
```